<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPopup</class>
 <widget class="QDialog" name="DialogPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blyat!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Icons/Nuclear.png</normaloff>:/Icons/Nuclear.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogPopup{
	border-image: url(:/Images/BackgroundError2.png)  0 0 0 0 stretch stretch;
}

#TextEditMessage{
	background-color: transparent;
	border: transparent;
	font: 20px 'EngraversGothic BT';
	color: white;
}

QToolButton{
	background-color: rgba(45,45,45, 80);
	border: transparent;
	font: 45px 'EngraversGothic BT';
	font-weight: bold;
	color: white;
}</string>
  </property>
  <widget class="QTextEdit" name="TextEditMessage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>460</width>
     <height>170</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonYes">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>230</y>
     <width>349</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Sim</string>
   </property>
  </widget>
  <widget class="QToolButton" name="ButtonNo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>349</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>NÃ£o</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ButtonYes</tabstop>
  <tabstop>ButtonNo</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
